presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlIntegrationSmokeTest.java

```diff
@@ -110,6 +110,19 @@ public void testInsert()
         assertUpdate("DROP TABLE test_insert");
     }
 
+    @Test
+    public void testInsertInPresenceOfNotSupportedColumn()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.not_supported_column_present (x bigint, y decimal(50,0), z varchar(10))");
```

Comment:
- remove space after table name
- rename the table to sth like `test_insert_not_supported_column_present`